src/Page.js

```diff
@@ -797,8 +797,7 @@ class Page {
         .then(result => this.insertFooterFile(result))
         .then(result => Page.insertTemporaryStyles(result))
         .then(result => markbinder.resolveBaseUrl(result, fileConfig))
-        .then(result => fs.outputFileAsync(this.tempPath, result))
-        .then(() => markbinder.renderFile(this.tempPath, fileConfig))
+        .then(result => markbinder.renderFile(result, this.tempPath, fileConfig))
```

Comment:
Thanks! I've looked through the logic and removed `Page#tempPath` altogether.

Also, I've removed `context` as an argument for `_parse` in `parser.js`, since `_parse` just passes around `context` without doing anything to it.